api/config/config.go

```diff
@@ -296,8 +296,10 @@ type StandardTransformerConfig struct {
 	EnableAuth            bool                 `envconfig:"FEAST_AUTH_ENABLED" default:"false"`
 	FeastRedisConfig      *FeastRedisConfig    `envconfig:"FEAST_REDIS_CONFIG"`
 	FeastBigtableConfig   *FeastBigtableConfig `envconfig:"FEAST_BIG_TABLE_CONFIG"`
+	FeastGPRCConnCount    int                  `envconfig:"FEAST_GRPC_CONN_COUNT" default:"10"`
 	FeastServingKeepAlive *FeastServingKeepAliveConfig
 	ModelClientKeepAlive  *ModelClientKeepAliveConfig
+	ModelServerConnCount  int `envconfig:"MODEL_SERVER_CONN_COUNT" default:"5"`
```

Comment:
Why do you have different default number of connection with FEAST_GRPC_CONN_COUNT ?

Reply:
for `FEAST_GRPC_CONN_COUNT` I used 10 because usually we have 10 feast istio pods (so it can spread evenly), and historically we faced several issues with imbalance load. No strong reason to use 5 for `MODEL_SERVER_CONN_COUNT`, just try to use smaller number for starter